courses/models.py

```diff
@@ -594,6 +598,17 @@ def __str__(self):
             )
         )
 
+    def save(self, *args, **kwargs):
+        if not self.certificate_page_revision:
+            certificate_page = (
+                self.course_run.course.page.certificate_page
+                if self.course_run.course.page
+                else None
+            )
+            if certificate_page:
+                self.certificate_page_revision = certificate_page.get_latest_revision()
+        super(CourseRunCertificate, self).save(*args, **kwargs)
+
```

Comment:
Can you add some test(s) for this?

Reply:
There are no tests added for this epic yet. I'll add the tests for all certificate apic with different PR